xml/issue4441.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version='1.0' encoding='utf-8' standalone='no'?>
+<!DOCTYPE issue SYSTEM "lwg-issue.dtd">
+
+<issue num="4441" status="New">
+<title><tt>ranges::rotate</tt> do not handle sized-but-not-sized-sentinel ranges correctly</title>
+<section>
+<sref ref="[alg.rotate]"/>
+<sref ref="[partial.sort]"/>
+<sref ref="[alg.nth.element]"/>
+<sref ref="[alg.merge]"/>
+</section>
+<submitter>Tomasz Kamiński</submitter>
+<date>03 Nov 2025</date>
+<priority>99</priority>
+
+<discussion>
+<b>Addresses US 161-258</b>
```

Comment:
```suggestion
<b><a href="https://github.com/cplusplus/nbballot/issues/833">Addresses US 161-258</a></b>
```

Reply:
These get turned into official ISO document (issue lists), so I am not sure if we want to link to closed github from there.

Reply:
I had the same concern, but we've been doing it for other NB comments in this ballot.

The issues list is not actually an official ISO doc any more. We stopped submitting it to ISO, it's just a WG21 doc now.